tests/e2e/integration_test.go

```diff
@@ -685,3 +687,26 @@ func (s *IntegrationTestSuite) waitForTransaction() {
 func TestIntegrationTestSuite(t *testing.T) {
 	suite.Run(t, new(IntegrationTestSuite))
 }
+
+func (s *IntegrationTestSuite) TestWeb3Sha3() {
```

Comment:
can you refactor the tests and use table tests instead?

Reply:
@fedekunze done, thanks for your advice